events/wmt.md

```diff
@@ -0,0 +1,103 @@
+---
+parent: Events
+title: WMT
+description: Conference on Machine Translation
+name: WMT
+---
+
+The Conference on Machine Translation, previously known as the Workshop on Machine Translation, **WMT**, is the main event for machine translation.
+
+During WMT's first years, the event held a series of workshops within the Empirical Methods in Natural Language Processing (EMNLP) conference organised by the Association for Computational Linguistics (ACL). In 2016, with the uprise of [neural machine translation](approaches/neural-machine-translation.md), WMT became a conference on its own, maintaining its name.
+
+> The conference aims to bring together academic scientists, researchers and industry representatives to exchange and share their experiences and research results. WMT plays a key role for the entire industry of computational linguistics and machine translation.
+
+## Shared tasks
+
+WMT hosts shared tasks competitions prior to the conference. During the shared tasks, researchers tackle specific challenges from a list of themes. Researchers then compare their techniques against the techniques of other researchers in the field.
+
+During the main conference, researchers present the results of the shared tasks and winners are announced.
+
+WMT started in 2006 with a *translation* task. In the following editions, WMT included themes on all aspects of machine translation, corpus preparation, training, and evaluation.
+
+### 2022 shared tasks
+
+- General machine translation task (former News task)
+- Biomedical translation task
+- Automatic post-editing task
+- Metrics task
+- Quality estimation task
+- Machine translation using terminologies task
+- Chat translation task
+- Sign language translation task
+- Robustness translation task
+- Triangual machine translation task
+- Large-scale multilingual machine translation task
+
+The results published from WMT shared tasks represent benchmarks for the technology.
+
+
+## Participants
+
+Universities and big tech companies consistently participate in the conference and are represented in the organising committee.
+
+### Participating universities
+
+*Incomplete list*
+
+- University of Edinburgh
+- Johns Hopkins University
+- Charles University
+- University of Amsterdam
+- University of Dublin
+- Fondazione Bruno Kessler
+- University of Amsterdam
+
+### Participating companies
+
+*Incomplete list*
+
+- IBM
+- Google
+- Amazon
+- Microsoft
+- Apple
+- Unbabel
+
+### Long-term organisers
+
+- [Philipp Koehn](community/people/philipp-koehn.md) (University of Edinburgh / Johns Hopkins University)
```

Comment:
Generally I don't think we should use parentheses for this, I'd prefer a comma or dash.

Also some of these people changed institutions or were at multiple institutions over the course of their involvement.